tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2978,6 +2978,31 @@ public function theActivitiesShouldBeInTheFollowingOrder(TableNode $table): void
 		}
 	}
 
+	/**
+	 * @When /^user "([^"]*)" lists the activities of (?:folder|file) "([^"]*)" from space "([^"]*)" with (depth|limit) "([^"]*)" and sort "([^"]*)" using the Graph API/
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $spaceName
+	 * @param string $filterType
+	 * @param string $filterValue
+	 * @param string $sortValue
+	 *
+	 * @return void
+	 */
+	public function userListsTheActivitiesForFolderOfSpaceWithDepthOrLimitAndSortUsingTheGraphApi(string $user, string $resource, string $spaceName, string $filterType, string $filterValue, string $sortValue): void {
```

Comment:
this is too long. please, break the line
```suggestion
	public function userListsTheActivitiesForFolderOfSpaceWithDepthOrLimitAndSortUsingTheGraphApi(
		string $user,
		string $resource,
		string $spaceName,
		string $filterType,
		string $filterValue,
		string $sortValue
	): void {
```